### ISSUE TITLE
Add permission shortcuts

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/next#356]
Enable/disable all permissions per collection, or per action

**Comment by Nitwel** [comment]
We could either make permissions selectable and have actions on the selected items or have icons next to the collection name to activate/deactivate the whole row. And for the columns, I don't really see the use of enabling a column.

**Comment by rijkvanzanten** [comment]
> And for the columns, I don't really see the use of enabling a column.

Enabling all read permissions at once is pretty useful!

**Comment by benhaynes** [comment]
I agree — enable/disable toggles for collection/row... no need for columns/permissions toggle.
